starter-code/index.html

```diff
@@ -1,42 +1,91 @@
+  <title>Spotify - Music for everyone</title>
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
-</body>
-</html>
+    <nav class="nav">
+      <div>
+        <img class="spotify-logo" src="../starter-code/images/spotify-logo.png" alt="spotify logo">
+      </div>
+      <div class="nav-links">
+        <a href="#" >Premium</a>
+        <a href="#" >Discover</a>
+        <a href="# ">Help</a>
+        <a href="#" >Download</a>
+      </div>
+    </nav>
+  
+    <div id="concert">
+      <h1>Music for everyone.</h1>
+        <p class="landingP">
+          Spotify is now free on mobile, tablet and computer.<br/>
+          Listen to the right music, wherever you are.
+        </p>
+    </div>
+  
+    <div id="container">
+      <div id="spot">
+        <h3>What’s on Spotify?</h3>
+        
+          <div id="music">
+            <img src="../starter-code/images/music-icon.png" alt="speaker">
+            <h4>Millions of Songs</h4>
+              <p>
+                There are millions of<br/>
+                songs on Spotify
+              </p>
+          </div>
+        
+          <div id="HQ">
+            <img src="../starter-code/images/high-quality-icon.png" alt="high quality">
+            <h4>HD Music</h4>
+              <p>
+                Listen to music as if you<br/>
+                were listening live
+              </p>
+          </div>
+        
+          <div id="stream">
+            <img src="../starter-code/images/devices-icon.png" alt="mobile devices">
+            <h4>Stream Everywhere</h4>
+              <p>
+                Stream music on your<br/>
+                smartphone, tablet or<br/>
+                computer
+              </p>
+          </div>
+      </div>
+    
+      <div id="yeezy">
+       <div id="kanye">
+          <h4 class="yeezyh4">It’s as yeezy as Kanye West.</h4>
+            <h4 class="kanyeh4">Search</h4>
+              <p>
+                Know what you want to listen to?<br/>
+                Just search and hit play.
+              </p>
+            <h4 class="kanyeh4">Browse</h4>
+              <p>
+                Check out the latest charts,<br/>
+                brand new releases and great<br/>
+                playlists for right now.
+              </p>
+            <h4 class="kanyeh4">Discover</h4>
```

Comment:
I would not write something like `"kanyeh4"` ... adding the `h4` part to the class, is what I don't like. Choose a better name, something like: `"kanye-title"`